api/fhda_api_1.1.yaml

```diff
@@ -0,0 +1,382 @@
+swagger: '2.0'
+info:
+  description: Initial API for FHDA
+  version: "1.1.1"
+  title: FHDA API
+
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+
+# tags are used for organizing operations
+tags:
+- name: admins
+  description: Secured Admin-only calls
+
+paths:
+  /course_by_quarter:
+    get:
+      tags:
+      - admins
+      summary: get course info
+      operationId: getCourseByQuarter
+      description: |
+        Search for course by quarter
+      produces:
+      - application/json
+      parameters:
+      - in: query
+        name: quarter
+        description: the quarter for searching
+        type: string
+
+      responses:
+        200:
+          description: search results matching criteria
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/CourseAcctD001A'
+        400:
+          description: bad input parameter
+          
+  /course_by_quarter_and_dept:
+    get:
+      tags:
+      - admins
+      summary: get course info
+      operationId: getCourseByQuarterAndDept
+      description: |
+        Search for course by inputing both quarter and department name
+      produces:
+      - application/json
+      parameters:
+      - in: query
+        name: quarter
+        description: the quarter for searching
+        type: string
+      - in: query
+        name: dept
+        description: the department name for searching
+        type: string
+
+      responses:
+        200:
+          description: search results matching criteria
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/CourseAcctD001A'
+        400:
+          description: bad input parameter
+          
+  /get_dept:
+    get:
+      tags:
+      - admins
+      summary: get deptartment info
+      operationId: getDept
+      description: |
+        Search for department given input department name
+      produces:
+      - application/json
+      parameters:
+      - in: query
+        name: deptName
+        description: the department name for searching
+        type: string
+      responses:
+        200:
+          description: search results matching criteria
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/DepartmentAcct'
+        400:
+          description: bad input parameter
+          
+  /add_course:
```

Comment:
Would it be better to use `/api/course/add`?

Reply:
Good point, will work on it in next commit